pyproject.toml

```diff
@@ -55,7 +55,6 @@ optional-dependencies.test = [
   "pytest-mock>=3.12",
   "virtualenv<21,>=20.25",
 ]
-urls.Changelog = "https://github.com/tox-dev/pipdeptree/blob/main/CHANGES.md"
```

Comment:
You should point to the GitHub releases page instead.

Reply:
Done.